src/services/localesServices/locales/en.json

```diff
@@ -125,5 +124,22 @@
 	"that you requested has been": "that you requested has been",
 	"To view this %s request you made click below": "To view this %s request you made click below",
 	"To view this edited request click below": "To view this edited request click below",
-	"request with id %s has been edited": "request with id %s has been edited"
+	"request with id %s has been edited": "request with id %s has been edited",
+	"Your manager posted a comment to your request": "Your manager posted a comment to your request",
+	"To view this As a your manager I can add any comment on your request. I can also cancel or approve your request but let me comment first request you made click below": "To view this As a your manager I can add any comment on your request. I can also cancel or approve your request but let me comment first request you made click below",
+	"View comment": "View comment",
+	"relation \"Users\" does not exist": "relation \"Users\" does not exist",
+	"To view this thanks for your comment manager. This is my request too. And please Approve my request after this comment request you made click below": "To view this thanks for your comment manager. This is my request too. And please Approve my request after this comment request you made click below",
+	"To view this yeahahaha 252525626262 request you made click below": "To view this yeahahaha 252525626262 request you made click below",
+	"As a your manager I can add any comment on your request. I can also cancel or approve your request but let me comment first": "As a your manager I can add any comment on your request. I can also cancel or approve your request but let me comment first",
+	"thanks for your comment manager. This is my request too. And please Approve my request after this comment": "thanks for your comment manager. This is my request too. And please Approve my request after this comment",
+	"your manager posted a comment": "your manager posted a comment",
+	"your requester posted a comment": "your requester posted a comment",
+	"To view this yeahahaha 252525626262 click below": "To view this yeahahaha 252525626262 click below",
+	"null value in column \"managerId\" violates not-null constraint": "null value in column \"managerId\" violates not-null constraint",
+	"To view this thanks for your comment manager. This is my request too. And please Approve my request after this comment click below": "To view this thanks for your comment manager. This is my request too. And please Approve my request after this comment click below",
```

Comment:
could you remove comments that are not used. I think they may have been generated when testing.

Reply:
sure

Reply:
done